src/NuGet.Core/NuGet.Frameworks/Strings.Designer.cs

```diff
@@ -8,7 +8,13 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace NuGet.Frameworks {
+//////////////////////////////////////////////////////////
+// Start - Chocolatey Specific Modification
+//////////////////////////////////////////////////////////
+namespace Chocolatey.NuGet.Frameworks
```

Comment:
You have removed the needed curly brace

```suggestion
namespace Chocolatey.NuGet.Frameworks {
```